package.json

```diff
@@ -63,6 +64,7 @@
     "js-yaml": "^3.4.6",
     "jsonfile": "^2.4.0",
     "kalabox-stats-client": "^0.2.1",
+    "linus": "git+https://github.com/tomas/linus.git#v0.0.5",
```

Comment:
This util looks handy! It has some pretty low usage numbers though. @pirog would you want to switch to something like this and add a dependency or just keep doing it ourselves?

Reply:
I wish they had a sync method.  That would definitely help due to the current use case.